drivers/gles3/rasterizer_storage_gles3.cpp

```diff
@@ -2186,8 +2186,8 @@ _FORCE_INLINE_ static void _fill_std140_variant_ubo_value(ShaderLanguage::DataTy
 
 			gui[0] = v.elements[0][0];
 			gui[1] = v.elements[0][1];
-			gui[2] = v.elements[1][0];
-			gui[3] = v.elements[1][1];
+			gui[4] = v.elements[1][0];
```

Comment:
Could you amend the commit and add comments explaining why it's 4,5 and not 2,3?

Reply:
Alright, should be a bit clearer now.